### PR TITLE
Creating a PR temeplate

### PR DESCRIPTION
**Pull request by StevenCopeland**
This template will allow people to view all requirements for the files and(hopefully) more people will obey them.

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+```
+ #### Name: [YOUR NAME](GitHub Link)
+  - Place: City, State, Country
+  - Bio: Who are you?
+  - GitHub: [Your Name](GitHub Link)
+```
+
+### If you are submiting a profile:
+
+ake sure to include your file in the Profile directory NOT(as I said before) in the home directory.
```
Comment (shahdivyam) [inline PULL_REQUEST_TEMPLATE.md]: *Make sure

**Comment by StevenCopeland** [comment]
@shahdivyam Thanks for catching that. Anything else?

**Comment by shahdivyam** [comment]
Looks good!

**Comment by Roshanjossey** [comment]
Hi  @StevenCopeland, I'm working on a similar project to help beginners to get started with contributing to open source projects.

Do check it out https://github.com/Roshanjossey/first-contributions

**Pull request merged**